### DISCUSSION TITLE
Problematic completion Org mode commands

### DISCUSSION THREAD

**Discussion by yantar92** [minad/vertico#459] [General]
I believe that the problematic Org mode commands are fixed on the latest main, after https://git.savannah.gnu.org/cgit/emacs/org-mode.git/commit/?id=5fa0f0c6fe3c0b1792821d976222b7de17cf9979

May you please confirm this?

**Comment by minad** [comment]
Ihor Radchenko ***@***.***> writes:

> I believe that the problematic Org mode commands are fixed on the latest main, after https://git.savannah.gnu.org/cgit/emacs/org-mode.git/commit/?id=5fa0f0c6fe3c0b1792821d976222b7de17cf9979
>
> May you please confirm this?

Thanks, Ihor! That's great. I am running Org from the main branch. I
will check.

**Comment by minad** [comment]
Ihor Radchenko ***@***.***> writes:

> I believe that the problematic Org mode commands are fixed on the latest main, after https://git.savannah.gnu.org/cgit/emacs/org-mode.git/commit/?id=5fa0f0c6fe3c0b1792821d976222b7de17cf9979

I've got this error:

In org-olpath-completing-read:
org-refile.el:778:30: Warning: reference to free variable ‘action’
org-refile.el:781:39: Warning: ‘match-beginning’ called with 2 args, but requires 1
org-refile.el:781:39: Warning: ‘match-beginning’ called with 2 arguments, but accepts only 1
org-refile.el:784:33: Warning: ‘match-beginning’ called with 2 args, but requires 1
org-refile.el:784:33: Warning: ‘match-beginning’ called with 2 arguments, but accepts only 1
libgccjit.so: error: gcc_jit_context_new_call_through_ptr: too many arguments to fn_ptr: freloc->R6d617463682d626567696e6e696e67_match_beginning_0 (got 2 args, expected 1)

**Comment by yantar92** [comment]
Daniel Mendler ***@***.***> writes:
> Ihor Radchenko ***@***.***> writes:
>
>> I believe that the problematic Org mode commands are fixed on the latest main, after https://git.savannah.gnu.org/cgit/emacs/org-mode.git/commit/?id=5fa0f0c6fe3c0b1792821d976222b7de17cf9979
>
> I've got this error:
>
> In org-olpath-completing-read:
> ...

That's silly copy-paste. Should be fixed now.

-- 
Ihor Radchenko // yantar92,
Org mode contributor,
Learn more about Org mode at <https://orgmode.org/>.
Support Org development at <https://liberapay.com/org-mode>,
or support my work at <https://liberapay.com/yantar92>

**Comment by minad** [comment]
@yantar92 `org-refile` does not yet work correctly with stepwise completion. There the boundaries should be similar to stepwise file path completion.

~~~ elisp
(setq org-refile-use-outline-path 'file
      org-outline-path-complete-in-steps t)
~~~
